### ISSUE TITLE
Import APK file directly from Play Store

### ISSUE THREAD

**Issue by StevenArzt** [secure-software-engineering/CodeInspect#146]
It would be good to have an import option for directly obtaining APK files from the Google Play store. Instead of asking for a file name, the import wizard should then ask for a package name and use existing download services for getting the respective APK.

**Comment by MarcMil** [comment]
We could use Racoon (Apache License): https://github.com/onyxbits/Raccoon
However, the user will need to supply credentials for his Google Account, since this is mandatory for downloading the apk file.
Another possibility is trying to include a wrapper which uses a website like Evozi's apk downloader, but this relies on the third party site, which does not seem like a good idea.
Personally I would go for the first approach.

**Comment by StevenArzt** [comment]
I agree that a library such as Raccoon would probably be the best solution. Relying on an external service that may change or stop working at any time always includes a risk.

The can then specify his Google account (or some test account) as soon as he tries to perform an APK download. We should also have a function to optionally save the credentials in Eclipse's key store for future use.
